### DISCUSSION TITLE
firefox: no audio with bluez-alsa on NixOS (dbus-system)

### DISCUSSION THREAD

**Discussion by imxade** [netblue30/firejail#5328] [Q&A] [answered]
On **NixOS**,
im using **bluez-alsa** for bluetooth headset

```
firejail --noprofile firefox
```
plays audio through default Bluetooth device

but 
```
firejail firefox
```
doesn't play audio when default device is a Bluetooth one.

i've whitelisted $HOME & $XDG_RUNTIME_DIR but no luck.

In case of **pulseaudio, firejailed firefox** plays fine through bluetooth.

how can i get it working with bluez-alsa ?

**Comment by kmk3** [comment, marked as answer]
Moved to:

* #5329
